packages/engine/errors/1163.md

```diff
@@ -0,0 +1,6 @@
+---
+original: "A 'yield' expression is only allowed in a generator body."
+excerpt: "Yield expressions must be used within a generator function"
+---
+
+If you want to use the functionality provided by yield, you should turn the enclosing function into a generator by adding an asterisk (*) next to the function name. Otherwise, you can remove the yield statement altogether.
```

Comment:
Could you add an example usage?

Also, `the functionality provided by yield`, `enclosing function`, `yield statement` can be turned into friendlier language.